### ISSUE TITLE
Porting to the Flysky i6s

### ISSUE THREAD

**Issue by Denniso2** [OpenI6X/opentx#386]
Is it possible to port this build to the flysky i6s? They two controllers use the exact same processor but the i6s has a touchscreen instead.
I have attached below some images of the internals.

![foto_no_exif](https://github.com/OpenI6X/opentx/assets/42325785/0ee77131-b7b9-4804-8d78-3f06b99ae39a)
![foto_no_exif (1)](https://github.com/OpenI6X/opentx/assets/42325785/622280e9-bc41-473f-a4ec-3c959fcc4467)

**Comment by ajjjjjjjj** [comment]
Yes. If you are a programmer you can contact me on discord regarding details.

**Comment by Denniso2** [comment]
I am but I do not have experience with opentx. I will hit you up on discord.

**Comment by wimalopaan** [comment]
Any progress here?

**Comment by ajjjjjjjj** [comment]
None AFAIK, but i'm willing to help even when non working PR will be created with added i2c touch driver. 
Buttons can be remapped to touch regions.
